packages/playkit-js-scripts/bin/kcontrib.js

```diff
@@ -25,13 +25,9 @@ let script = scriptIndex === -1 ? args[0] : args[scriptIndex];
 const nodeArgs = scriptIndex > 0 ? args.slice(0, scriptIndex) : [];
```

Comment:
General comment - `package.json` still contain dependencies of `webpack`

 "webpack": "4.41.2",
    "webpack-cli": "3.3.10",
    "webpack-dev-server": "3.9.0",
    "webpack-merge": "4.2.2"

source-map-loader
html-webpack-plugin
css-loader
clean-webpack-plugin
copy-webpack-plugin

etc...

Reply:
I will remove them in FEV-439 when we gonna move all webpack-dependent stuff to cli